sdk/ext/ovirtsdk4c/ov_http_client.c

```diff
@@ -490,12 +493,20 @@ static int ov_http_client_debug_function(CURL* handle, curl_infotype type, char*
     /* Get the pointer to the transfer: */
     ov_http_transfer_ptr(transfer, transfer_ptr);
 
-    /* Execute the debug code with the global interpreter lock acquired, as it needs to call Ruby methods: */
+    /* The global interpreter lock may be acquired or not, so we need to check and either call the task directly
+       or else call it after acquiring the lock. Note that the `ruby_thread_has_gvl_p` function that we ue to
```

Comment:
```suggestion
       or else call it after acquiring the lock. Note that the `ruby_thread_has_gvl_p` function that we use to
```

Reply:
ooh, I see this commit in the other PR, will bring it across